pkg/portutil/portutil.go

```diff
@@ -100,7 +100,15 @@ func ParseFlagP(s string) ([]gocni.PortMapping, error) {
 		if err != nil {
 			return nil, fmt.Errorf("invalid hostPort: %s", hostPort)
 		}
+		usedPorts, err := getUsedPorts(ip, "tcp")
+		if err != nil {
+			return nil, err
+		}
+		if usedPorts[startHostPort] {
```

Comment:
why checking only  `startHostPort` ?  why not testing all the range ?

Reply:
Good point, updated.